front/src/applications/editor/components/LinearMetadata/FormComponent.tsx

```diff
@@ -427,4 +423,54 @@ export const FormComponent: React.FC<FieldProps> = (props) => {
   );
 };
 
+export const FormComponent: React.FC<FieldProps> = (props) => {
+  const { name, formContext, schema, registry } = props;
+  const Fields = utils.getDefaultRegistry().fields;
+
+  // Get the distance of the geometry
+  const distance = useMemo(() => {
+    if (!isNil(formContext.length)) {
+      return formContext.length;
```

Comment:
Add a cast here to specify distance type
```suggestion
      return formContext.length as number;
```